olad/OladHTTPServer.cpp

```diff
@@ -219,6 +219,155 @@ OladHTTPServer::OladHTTPServer(ExportMap *export_map,
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-icon-180x180.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-icon-57x57.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-icon-60x60.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-icon-72x72.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-icon-76x76.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-icon-precomposed.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-icon.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-startup-image-1182x2208.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-startup-image-1242x2148.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-startup-image-1496x2048.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-startup-image-1536x2008.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-startup-image-320x460.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-startup-image-640x1096.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-startup-image-640x920.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-startup-image-748x1024.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-startup-image-750x1294.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/apple-touch-startup-image-768x1004.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/browserconfig.xml",
+      HTTPServer::CONTENT_TYPE_XML);
+  m_server.RegisterFile(
+      "/new/img/favicons/coast-228x228.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/favicon-16x16.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/favicon-230x230.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/favicon-32x32.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/favicon-96x96.png",
+      HTTPServer::CONTENT_TYPE_PNG);
+  m_server.RegisterFile(
+      "/new/img/favicons/favicon.ico",
+      HTTPServer::CONTENT_TYPE_OCT);
```

Comment:
https://en.wikipedia.org/wiki/ICO_%28file_format%29#MIME_type and http://stackoverflow.com/questions/13827325/correct-mime-type-for-favicon-ico suggest a MIME type of image/x-icon should be correct, would you mind changing to that please @dandaka ?